app-backend/api/src/main/scala/user/Routes.scala

```diff
@@ -42,6 +51,11 @@ trait UserRoutes extends Authentication
       pathEndOrSingleSlash {
         get { getUserByEncodedAuthId(authIdEncoded) } ~
         put { updateUserByEncodedAuthId(authIdEncoded) }
+      } ~
+      pathPrefix("dropbox-setup") {
```

Comment:
If we are removing the user id from the path it might as well be its own endpoint not tied to /users at all so as not to be confusing. What do you all think?